### ISSUE TITLE
Error Compiling 

### ISSUE THREAD

**Issue by Immortal-** [omnicoinsource/Omnicoin#4]
src/allocators.h:10:34: fatal error: boost/thread/mutex.hpp: No such file or directory
 #include <boost/thread/mutex.hpp>
                                  ^
compilation terminated.
make: **\* [build/bitcoin.o] Error 1

Omnicoin$ lsb_release -irc
Distributor ID: Ubuntu
Release:    13.10
Codename:   saucy

**Comment by SteenSchutt** [comment]
Which steps did you do to compile it?

**Comment by Immortal-** [comment]
I got this after using the `make` command, after using the `qmake "USE_UPNP=-"` command

**Comment by iangcarroll** [comment]
Are you trying to compile QT or OmniCoind?

**Comment by SteenSchutt** [comment]
I think it's safe to assume that he's attempting to compile the QT wallet since he's running qmake.
Your error seems to indicate that boost-thread is missing. There can be several reasons for that.
1. You didn't install libboost-threads-dev (or libboost-all-dev which references this among others)
2. Ubuntu 13 doesn't have the new version of libboost in the repos and will need to be referencing libboost-threads-mt instead.

Try running `apt-cache showpkg libboost-thread-dev` and post the output. Root permissions not needed.

**Comment by Immortal-** [comment]
yea It looks like i just didnt have the library installed stupid me.

**Comment by caffeinewriter** [comment]
Yeah, sometimes Boostlib installs without certain components. No clue why, but it seems to happen with certain repos. `apt-get install libboost-all-dev` usually works. Regardless, glad you got it fixed.
